### DISCUSSION TITLE
Contraindication to the use of a root domain for the return path (RP)?

### DISCUSSION THREAD

**Discussion by dtrckd** [postalserver/postal#2405] [Help with using Postal]
Fom the [doc](https://docs.postalserver.io/install/dns-configuration) we can learn that the RP is used as the MAIL FROM for all messages sent through a mail server. Additionally, when I setup postal for the domain let's say `mydomain.com`, it asks me to set a DNS record like this

| Hostname | Type | Value |
|--------|--------|--------|
| psrp.mydomain.com  | CNAME |  rp.postal.mydomain.com | 

Everything is okay, except when I try to use **smtp_relays** to forward my emails to an external SMTP server to relay their sends. The problem encountered here is that the external SMTP complains that ` "From:" header does not match the sender's domain`  

Indeed the external SMTP server has to be configured to accept/relay email from psrp.mydomain.com, but as we send email like <joe@mydomain.com>, both domain does not match.

So, to solve this, I am wondering if the return path could just be equal to the root domain (`mydomain.com` in the example) or if there is issues with that ?

**Comment by dtrckd** [comment]
could be related to https://github.com/postalserver/postal/issues/255

**Comment by willpower232** [comment]
I'm not fully sure but is this what the sender header is supposed to resolve?

**Reply by dtrckd** [reply]
Do you refer to the `sender` field mentioned here : https://apiv1.postalserver.io/controllers/send/message.html

I will try to set it and see if it solves the issue.
I let you know.
